src/lib.rs

```diff
@@ -114,24 +114,22 @@
 //! # use std::convert::TryFrom;
 //! # use rand::rngs::OsRng;
 //! # use std::convert::TryInto;
-//! # use ed25519_dalek::{Keypair, Signature, Signer, PublicKey, SecretKey, SignatureError};
+//! # use ed25519_dalek::{SigningKey, Signature, Signer, VerifyingKey, SecretKey, SignatureError};
 //! # use ed25519_dalek::{PUBLIC_KEY_LENGTH, SECRET_KEY_LENGTH, KEYPAIR_LENGTH, SIGNATURE_LENGTH};
-//! # fn do_test() -> Result<(SecretKey, PublicKey, Keypair, Signature), SignatureError> {
+//! # fn do_test() -> Result<(SigningKey, VerifyingKey, Signature), SignatureError> {
 //! # let mut csprng = OsRng{};
-//! # let keypair_orig: Keypair = Keypair::generate(&mut csprng);
+//! # let signing_key_orig: SigningKey = SigningKey::generate(&mut csprng);
 //! # let message: &[u8] = b"This is a test of the tsunami alert system.";
-//! # let signature_orig: Signature = keypair_orig.sign(message);
-//! # let public_key_bytes: [u8; PUBLIC_KEY_LENGTH] = keypair_orig.public_key().to_bytes();
-//! # let secret_key_bytes: [u8; SECRET_KEY_LENGTH] = keypair_orig.secret_key().to_bytes();
```

Comment:
So there used to be a way to get a `SecretKey` from a `Keypair` and now there isn't? This seems fine. Is there a particular reason that was there in the first place?